source/manual/how-to-be-a-good-buddy.html.md

```diff
@@ -0,0 +1,40 @@
+---
+owner_slack: "#govuk-developers"
+title: How to be a good buddy
+section: Learning GOV.UK
+layout: manual_layout
+type: learn
+parent: "/manual.html"
+---
+
+When a new joiner starts on GOV.UK, it's good practice to assign them a "buddy" to help them get started on GOV.UK, explain it's infrastructure and provide initial support.
+
+## What is a buddy?
+
+A buddy acts as a temporary guide for new joiners. Their responsibilities include:
```

Comment:
```suggestion
A buddy acts as a temporary guide for new joiners. It's an informal role and their responsibilities include:
```

I kind of wanted to get the phrase "informal role" in somewhere and thought this was best place.